src/web/components/Core/Dialog.tsx

```diff
@@ -31,7 +31,7 @@ export function Dialog({
       <RadixDialog.Content
         className='dialog-container'
         // To prevent dialog closed from user clicking outside of the dialog
-        onInteractOutside={(fireEvent) => fireEvent.preventDefault()}
+        onInteractOutside={(fireEvent) => fireEvent.preventDefault()} // eslint-disable-line @typescript-eslint/no-unsafe-call
```

Comment:
I'm not getting a lint rule violation either in VS Code or `npm run lint` - is this needed?

Reply:
oh yeah! I swear I was getting a big black overlay before, but can't seem to reproduce anymore. Removed ✅